DNN Platform/Website/Default.aspx.cs

```diff
@@ -672,89 +687,98 @@ private string GenerateCssCustomProperties()
                     return $$"""
                              <style type="text/css">

```

Comment:
Should we consdier adding a nonce or hash to this inclusion to help with CSP management for those looking to set security policies?

Reply:
Sure I'll look into that next time I can